gpu-operator/life-cycle-policy.rst

```diff
@@ -141,12 +143,20 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
    * - NVIDIA GDRCopy Driver
      - `v2.5.0 <https://github.com/NVIDIA/gdrcopy/releases>`__
 
-.. _gds-open-kernel:
+.. _known-issue:
 
    :sup:`1`
+   Known Issue: For drivers 570.124.06, 570.133.20, and 570.148.08, 
+   GPU workloads cannot be scheduled on nodes that have a mix of MIG slices and full GPUs. 
+   This manifests as GPU pods getting stuck indefinitely in the ``Pending`` state. 
+   The solution is to downgrade the driver to version 570.86.15 to work around this issue.
```

Comment:
```suggestion
   It's recommended that you downgrade the driver to version 570.86.15 to work around this issue.
```

just a nit suggestions. I think the "it's recommended" wording used in other places in the site, so i thought it might help add some consistency